clients/java/src/main/java/io/zeebe/client/impl/Loggers.java

```diff
@@ -24,4 +24,5 @@
       LoggerFactory.getLogger("io.zeebe.client.job.poller");
   public static final Logger JOB_WORKER_LOGGER =
       LoggerFactory.getLogger("io.zeebe.client.job.worker");
+  public static final Logger VERSION_LOGGER = LoggerFactory.getLogger("io.zeebe.client.util");
```

Comment:
The logger package looks a bit uncommon with `.util`. Maybe, we can have a logger for general or managing things `io.zeebe.client`.